client/readme.md

```diff
@@ -1 +1,65 @@
-## React-app
+<div align="center">
```

Comment:
this is already on the root directory. if you want to add client specific readme, create something unique and more specific to the client, or take this out.

Reply:
Removed this from client directory